src/helpers.js

```diff
@@ -0,0 +1,55 @@
+const sha1 = require('sha1');
+
+export const checksum = str => parseInt(sha1(str).substr(0, 8), 16);
+
+export const getLastItem = obj => {
+  const keys = Object.keys(obj);
+  const lastKey = keys[keys.length - 1];
+
+  const item = obj[lastKey];
+  delete obj[lastKey];
+
+  return item;
+};
+
+export const getSql = () => {
+  let sql;
+  if (process.env.APP_ENV === 'browser') {
+    require('script!sql.js');
+    sql = window.SQL;
+  } else {
+    sql = require('sql.js');
+  }
+  return sql;
+};
+
+export const rand = () => Math.random() * 100000000 | 0;
+
+export const getDb = () => new (getSql().Database)();
+
+export const getZip = (...args) => {
```

Comment:
This is how all JavaScript modules I've ever encountered written (hard-code dependencies overridable by external tools like `proxyquire`). Perhaps, if you show some more repositories with such style of testing (every external dependency wrapped in a helper function and all of them passed in an entry point), I will be able to see the benefits.